rtp-session.go

```diff
@@ -81,6 +81,25 @@ func (s *RTPSession) Decode(packetData []byte) (*RTPPacket, error) {
 	return p, nil
 }
 
+func (s *RTPSession) DecodeRTCP(packetData []byte) (*RTCPCompoundPacket, error) {
+
+	p, err := NewSRTCPPacket(packetData)
+	if err != nil {
+		return nil, err
+	}
+
+	if s.cipher != NONE {
+		err = p.DecryptGCM(s.key, s.salt)
```

Comment:
This is still incorrect: https://tools.ietf.org/html/rfc3711#section-4.3.2

You need to add two members `rtcpKey` and `rtcpSalt` on `RTPSession` and update `RTPSession.SetSRTP` to derive them.